BackofficeBundle/Form/Type/Component/ContentSearchType.php

```diff
@@ -10,6 +10,16 @@
  */
 class ContentSearchType extends AbstractType
 {
+    protected $transformer;
+
+    /**
+     * @param string $transformerClass
+     */
+    public function __construct($transformerClass)
+    {
+        $this->transformerClass = $transformerClass;
```

Comment:
Why not use class_exists($transformerClass) to check that the parameter is a string, a valid namespace and mapping a known class ?